doc/changes/devel/12523.bugfix.rst

```diff
@@ -0,0 +1 @@
+Remove FDT file format check for strings in EEGLAB's EEG.data in :func:_check_eeglab_fname by :newcontrib:`Seyed Yahya Shirazi`
```

Comment:
The release notes are meant for public consumption so we should only refer to public names/attributes
```suggestion
Remove FDT file format check for strings in EEGLAB's EEG.data in :func:`mne.io.read_raw_eeglab` and related functions by :newcontrib:`Seyed Yahya Shirazi`
```